integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/wait/service/database/DatabaseAwait.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.it.cloudbreak.util.wait.service.database;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.it.cloudbreak.RedbeamsClient;
+import com.sequenceiq.it.cloudbreak.context.RunningParameter;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.database.RedbeamsDatabaseServerTestDto;
+import com.sequenceiq.it.cloudbreak.log.Log;
+import com.sequenceiq.it.cloudbreak.util.ResponseUtil;
+import com.sequenceiq.it.cloudbreak.util.wait.service.Await;
+import com.sequenceiq.it.cloudbreak.util.wait.service.ExceptionChecker;
+import com.sequenceiq.it.cloudbreak.util.wait.service.Result;
+import com.sequenceiq.it.cloudbreak.util.wait.service.WaitResult;
+import com.sequenceiq.redbeams.api.model.common.Status;
+
+@Component
+public class DatabaseAwait extends Await<RedbeamsDatabaseServerTestDto, Status> {
```

Comment:
I would rename Database* to Redbeams*, to avoid confusion with the existing Database endpoints on cloudbreak.
Or, if you want to keep the database name, then I would rename it to DatabaseServer*, to avoid confusion with Database endpoints on redbeams.